prdoc/pr_5376.prdoc

```diff
@@ -0,0 +1,3 @@
+crates:
```

Comment:
same with this file, it is coming from some other PR, I think something got messed up in your branch when you changed the base from master to xcmv5

Reply:
xcmv5 has been merged with latest master now, you can try merging